tests/ui/components/search/spellcheckcomponent.js

```diff
@@ -0,0 +1,58 @@
+import DOM from '../../../../src/ui/dom/dom';
+import mockManager from '../../../setup/managermocker';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import SpellCheckComponent from '../../../../src/ui/components/search/spellcheckcomponent';
+import GlobalStorage from '../../../../src/core/storage/globalstorage';
+
+const mockCore = {
+  globalStorage: new GlobalStorage()
+};
+
+const COMPONENT_MANAGER = mockManager(mockCore);
+
+describe('spellcheck redirect links', () => {
```

Comment:
While this test coverage is definitely good to have, is it really related to Storage? If it is, the relationship feels tangential. Again, more test coverage is always better. This item is fairly large in terms of scope already. So, I think we need to limit ourselves to tests that verify interactions with storage.

Reply:
That's correct, these tests cover the encoding of persistent storage rather than a direct interaction with persistent storage. It was one of the use cases that Christian pointed out so I added some coverage, however I will stick to more direct usage of storage in the future